wagtail/admin/templates/wagtailadmin/generic/chooser/checkbox_select_cell.html

```diff
@@ -1,4 +1,4 @@
 {% load i18n %}
 <td {% if column.classname %}class="{{ column.classname }}"{% endif %}>
-    <input type="checkbox" id="chooser-modal-select-{{ value }}" data-multiple-choice-select name="id" value="{{ value }}" title="{% blocktrans trimmed with title=instance %}Select {{ title }}{% endblocktrans %}">
+    <input type="checkbox" id="chooser-modal-select-{{ value }}" data-multiple-choice-select name="id" value="{{ value|unlocalize }}" title="{% blocktrans trimmed with title=instance %}Select {{ title }}{% endblocktrans %}">
```

Comment:
The other occurrence of `value` (in `"chooser-modal-select-{{ value }}"`) needs updating too. Have also just noticed that there's another template that needs the same fixes - `wagtail/admin/templates/wagtailadmin/chooser/tables/page_checkbox_select_cell.html`

Reply:
Thanks for reviewing. I will make required changes.